runtime/compiler/build/rules/gnu/common.mk

```diff
@@ -47,7 +47,7 @@ endif
 jit: $(JIT_PRODUCT_SONAME)
 
 $(JIT_PRODUCT_SONAME): $(JIT_PRODUCT_OBJECTS) | jit_createdirs
-	$(SOLINK_CMD) -shared $(SOLINK_FLAGS) $(patsubst %,-L%,$(SOLINK_LIBPATH)) -o $@ $(SOLINK_PRE_OBJECTS) $(JIT_PRODUCT_OBJECTS) $(SOLINK_POST_OBJECTS) $(LINK_GROUP_START) $(patsubst %,-l%,$(SOLINK_SLINK)) $(LINK_GROUP_END) $(SOLINK_EXTRA_ARGS)
+	$(SOLINK_CMD) -shared $(SOLINK_FLAGS) $(patsubst %,-L%,$(SOLINK_LIBPATH)) -o $@ $(SOLINK_PRE_OBJECTS) $(JIT_PRODUCT_OBJECTS) $(SOLINK_POST_OBJECTS) $(LINK_GROUP_START) $(patsubst %,-l%,$(SOLINK_SLINK)) $(SOLINK_SLINK_STATIC) $(LINK_GROUP_END) $(SOLINK_EXTRA_ARGS)
```

Comment:
This is one place where the new code is not guarded by `ifneq ($(JITSERVER_SUPPORT),)`.
What are the implications of `$(SOLINK_SLINK_STATIC)` ?

Reply:
Trying to eliminate `JITSERVER_SUPPORT` flag when it's possible. $(SOLINK_SLINK_STATIC) is not used/defined in the baseline code. It is used for statically linking protobuf which is already guarded by `JITSERVER_SUPPORT` in ` build/toolcfg/gnu/common.mk` as below. 

```
ifneq ($(JITSERVER_SUPPORT),)
    #
    # Setup protobuf
    #
    PROTO_CMD?=protoc

    SOLINK_SLINK_STATIC=-l:libprotobuf.a
    CXX_DEFINES+=GOOGLE_PROTOBUF_NO_RTTI
```

Reply:
I've decided to guard it with `JITSERVER_SUPPORT`